src/core/AutoRest/Program.cs

```diff
@@ -91,13 +91,6 @@ private static int Main(string[] args)
                             return 0;
                         }
 
-                        // comparison?
```

Comment:
Oh. I thought we had at least moved to `-FANCY` ... I totally forgot about that. 

I haven't run with the legacy cmdline in a long time at all tho' :D

Ah, right, we have to write a better cmdline help before we turn off that.